ingest.py

```diff
@@ -76,7 +76,7 @@ def load(self) -> List[Document]:
     ".pdf": (PDFMinerLoader, {}),
     ".ppt": (UnstructuredPowerPointLoader, {}),
     ".pptx": (UnstructuredPowerPointLoader, {}),
-    ".txt": (TextLoader, {"encoding": "utf8"}),
+    ".txt": (TextLoader, {"encoding": "utf8", "autodetect_encoding": True}),
```

Comment:
`"encoding": "utf8"` should be removed in favour of the second parameter, right?

Reply:
The script tries to decode with "encoding" Parameter first,if it fails, uses autoencoding feature. With no encpding param, it defaults to the systeö encoding.